deploy/templates/bicep/devicecontent.json

```diff
@@ -0,0 +1,78 @@
+{
+    "templateType": "CollectionContent",
```

Comment:
Suggestion: Is there a way to resolve these from our https://github.com/microsoft/iomt-fhir/tree/main/sample/templates path?  Concern that having example templates here is another code location we will need to remember/maintain.

Reply:
Yes, I can remove these. I originally had these here for testing purposes and forgot to remove them.